src/ui/components/achievements/components/GenericAchievement.tsx

```diff
@@ -77,7 +77,7 @@ export const GenericAchievement: FunctionComponent<Props> = (props: Props) => {
               )}
             </Swiper>
           </SwiperContainer>
-          <Spacer.Column numberOfSpaces={getSpacing(2)} />
+          <Spacer.Column numberOfSpaces={getSpacing(0.3)} />
```

Comment:
```suggestion
          <Spacer.Column numberOfSpaces={getSpacing(1)} />
```
On est sensé n'utiliser que des nombres entiers non ? 
Car 0.3 * 4 dpi ca n'existe pas.

Reply:
Je pensais que `getSpacing` s'occupait de le rendre factorisable par 4. De toute évidence, le réglage des décimals réduit bien la taille de l'espacement.